django_readers/pairs.py

```diff
@@ -24,14 +25,20 @@ def combine(*pairs):
     return qs.pipe(*prepare_fns), projectors.combine(*project_fns)
 
 
-def prepare_only(prepare):
+def with_noop_projector(prepare):
     return prepare, projectors.noop
 
 
-def project_only(project):
+def with_noop_queryset_function(project):
     return qs.noop, project
```

Comment:
I'm not sure if the functions provide any extra value than just inlining the body at the call site, honestly 🤔

Reply:
You mean just putting `(qs.noop, whatever)` in your spec directly?

Reply:
I am leaning towards agreeing with this

Reply:
Done